Remove icons in standalone search

andvirga opened the pull request:
- Removed `MoreHorizIcon` (...) in all scenarios
- Removed `LinkIcon` for Standalone search scenario.
- Fixed Flex reboot issue by chaging `<form>` wrapper to a `<div>`

nickhurlburt reviewed (changes requested):
Mostly looking good, but some major issues.  Biggest thing is that whenever I do an in-contact search, then click into the cases search tab, Flex gets an error and forces a hard refresh.  Several errors and warnings get reported when this happens.  Please test out in-contact search to see this.

andvirga commented on plugin-hrm-form/src/components/ContactDetails.tsx:
```diff
@@ -1,8 +1,8 @@
+import { IconButton } from '@material-ui/core';
+import { Link as LinkIcon } from '@material-ui/icons';
 
 import { DetailsContainer, NameContainer, DetNameText, ContactDetailsIcon } from '../styles/search';
```
Good catch! Done!

andvirga commented on plugin-hrm-form/src/components/search/ContactPreview/ConnectContact.tsx:
```diff
@@ -27,7 +28,7 @@ const ConnectContact: React.FC<Props> = ({ callType, onOpenConnectDialog }) => {
           <Template code="ContactPreview-CopyButton" />
         </HiddenText>
-        <ConnectIcon />
+        {showConnectIcon && <ConnectIcon />}
```
Solved!

andvirga commented:
@nickhurlburt the flex reboot issue is solved now. (I've removed the `<form>` wrapper and replaced it with a `<div role='form'>` like @GPaoloni instructed.

nickhurlburt reviewed (approved):
Looks good to me!

murilovmachado reviewed (approved):
Looks good!
